keyboards/wolf/silhouette/rules.mk

```diff
@@ -0,0 +1 @@
+WS2812_DRIVER = vendor
```

Comment:
```suggestion
```

Reply:
Is vendor now the default?

Reply:
No, I don't see where the WS2812 driver is configured. You haven't defined a pin or enabled RGBlight/RGB Matrix.

Reply:
oh that's fair, sorry about that